resources/js/components/dashboard/DashboardModule.vue

```diff
@@ -2,35 +2,46 @@
-            </div>
-            <div class="dashboard-module__header">
-              {{item.name}}
-            </div>
-          </div>
-          <div class="row   dashboard-module__body align-items-center">
-            <div class="col-2 text-left">
-              <div class="dashboard-module__progress">
-                <p class="dashboard-module__progress-text"
-                :class="item.completion.progress.toString().length <= 2 ?
-                'dashboard-module__progress-text--sm' :
-                'dashboard-module__progress-text--lg'">{{ item.completion.progress }}%
-                </p>
-                <loading-progress class="dashboard-module__loader"
-                  :progress="item.completion.progress/100"
-                  :size="size"
-                  :shape="shape.circle"
-                />
+          <div class="row">
+            <div class="dashboard-module__overview-wrapper col-12 col-md-6">
+              <div class="dashboard-module__overview row">
+                <div class="dashboard-module__thumbnail">
+                  <img class="dashboard-module__thumbnail-image" v-bind:src="item.image" v-bind:alt="item.altText">
+                </div>
               </div>
             </div>
-            <div class="col-5 text-left">
-              <span class="dashboard-module__date ml-3">{{ item.completion.date}}</span>
-            </div>
-            <div class="col-4 text-right">
-              <span class="dashboard-module__status">{{item.completion.status}}</span>
-              <i v-bind:class="[ item.completion.completed ? 'dashboard-module__status-circle fas fa-check-circle' : 'fas fa-chevron-right' ]"></i>
+
+            <div class="dashboard-module__info col-12 col-md-6">
+              <div class="dashboard-module__header">
+                {{item.name}}
+              </div>
+              <div class="dashboard-module__description">
+                {{item.description}}
+              </div>
+
+              <div class="dashboard-module__body row align-items-center">
+                <div class="col-2 text-left">
+                  <div class="dashboard-module__progress">
+                    <p class="dashboard-module__progress-text"
+                    :class="item.completion.progress.toString().length <= 2 ?
+                    'dashboard-module__progress-text--sm' :
+                    'dashboard-module__progress-text--lg'">{{ item.completion.progress }}%
+                    </p>
+                    <loading-progress class="dashboard-module__loader"
+                      :progress="item.completion.progress/100"
+                      :size="size"
+                      :shape="shape.circle"
+                    />
+                  </div>
+                </div>
+                <div class="col-5 col-md-10 text-left">
+                  <span class="dashboard-module__date ml-3">{{ item.completion.date}}</span>
```

Comment:
based on the mockup it seems like you can add a `mt-1` class in this div to add some margin to the text that goes under here